src/bun.js/module_loader.zig

```diff
@@ -1789,6 +1789,17 @@ pub const ModuleLoader = struct {
                 }
 
                 if (loader == .json or loader == .toml) {
```

Comment:
Based on this branch, does this bug apply to loading empty `.json` files too? If so, should add a test?

Reply:
yes, but specifically `package.json` or `tsconfig.json`